mcs/class/corlib/Test/System.Reflection/FieldInfoTest.cs

```diff
@@ -224,6 +224,16 @@ public void GetCustomAttributes ()
 			Assert.AreEqual (typeof (ObsoleteAttribute), attrs [0].GetType (), "#D10");
 		}
 
+		[Test]
+		public void MetadataToken_DefinedInMonoField ()
+		{
+			Type type = typeof (FieldInfoTest);
+			FieldInfo field = type.GetField ("i");
+			PropertyInfo MetadataTokenProperty = field.GetType ().GetProperty ("MetadataToken");
+			Assert.AreEqual ("MonoField", MetadataTokenProperty.DeclaringType.Name);
```

Comment:
Why do we care where the property is declared?

Would it be enough just to test that `Assert.IsTrue ( ((int)field.MetadataToken) > 0);`  (ie it's some token - and implicitly - that using the property doesn't throw)?

Reply:
@lambdageek I thought that the above comment about overriding test required it.

Reply:
@lambdageek Okay, I'll simplify those tests, thanks!